gettsim/benefits/arbeitsl_geld.py

```diff
@@ -49,13 +49,13 @@ def proxy_net_wage_last_year(
     wage."""
 
     # Relevant wage is capped at the contribution thresholds
-    max_wage = min(beit_bem_grenz, person["m_wage_l1"])
+    max_wage = min(beit_bem_grenz, person["bruttolohn_vorj_m"])
 
     # We need to deduct lump-sum amounts for contributions, taxes and soli
     prox_ssc = soz_vers_pausch * max_wage
 
     # Fictive taxes (Lohnsteuer) are approximated by applying the wage to the tax tariff
-    prox_tax = e_st_params["tax_schedule"](12 * max_wage - werbungs_pausch, e_st_params)
+    prox_tax = e_st_params["st_tarif"](12 * max_wage - werbungs_pausch, e_st_params)
```

Comment:
Please change `e_st` to `eink_st` everyhwere, including the yaml file name etc. (Only leave ESt as the abbreviation of the law)

Reply:
Changed to `eink_st` everywhere!